public/site.webmanifest

```diff
@@ -1,10 +1,16 @@
 {
+  "short_name": "Lunch",
+  "name": "Lunch",
+  "description": "An app for groups to decide on nearby lunch options.",
```

Comment:
Could you use the description from `server.js`?

Reply:
Yes. Do you want me to change the description at the top of the README?